scripts/get-model-data.py

```diff
@@ -377,21 +484,37 @@
             required_keys = list(config['keys']) + list(config.get('assign_json_subfields', {}).keys())
             data = queries.base_model(config['base_model'], required_keys, config['get_data_kwargs'],
                                       client=client, logger=logger, limit=limit)
+
         elif 'model' in config:
             data = queries.model(config['model'], directory=OUTPUT_DIR)
+
         if 'joins' in config:
             for join in config['joins']:
                 data = queries.join(data, directory=OUTPUT_DIR, **join)
+
+        if 'get_by_model_fk' in config:
+            data = queries.get_by_model_fk(
+                config['get_by_model_fk'],
+                config['keys'],
+                data,
+                client
+            )
+
         # transform values that we want to transform
         if 'assign_json_subfields' in config:
             for field, subfields in config['assign_json_subfields'].items():
                 data = queries.assign_json_subfields(field, subfields, data)
+
         if 'duplicate_fields' in config:
             for field, new_name in config['duplicate_fields']:
                 data = queries.duplicate_fields(data, field, new_name)
 
+        if 'process_fields' in config:
```

Comment:
🤔 Iiiinnteresting, so this needs to come above group by because of the `convert_none_to_zero` fair enough

Reply:
Yep. The whole thing is a bit weird. It's this sort of declarative thing where you specify a config for the data you want, but you also need to keep in mind the order things are getting applied in. 

It'd be nice if there was some sort of list of actions, which were different queries to apply, and they got applied in order.